### PR TITLE
Handle polymorphic disjoint in cgal

### PR DESCRIPTION
**Pull request by pentacular**

### CODE REVIEW

File: geometry/tagged/disjoint.js [hunk 1]
```diff
@@ -1,10 +1,11 @@
 import { rewrite, visit } from './visit.js';
 
 import { cut } from './cut.js';
+import { disjoint as disjoinGraphs } from '../graph/disjoint.js';
 import { taggedGroup } from './taggedGroup.js';
 import { toConcreteGeometry } from './toConcreteGeometry.js';
 
-export const disjoint = (geometries) => {
+export const disjoint = (geometries, strategy = 'incremental') => {
```
Comment (BarbourSmith) [inline geometry/tagged/disjoint.js:8]: Is incremental the new one?

**Review by BarbourSmith — APPROVED** [review]
No big thoughts on the inner workings other than that I am excited to try it. I think using it in practice will be the real test so I'll update tomorrow if the tests pass and try to break things :p

File: geometry/tagged/disjoint.js (hunk 1 above)
Comment (pentacular) [reply geometry/tagged/disjoint.js:8]: Yeah, but you can switch back if it has problems.

Comment (BarbourSmith) [reply geometry/tagged/disjoint.js:8]: Great 👍, I just wanted to make sure I tested the right one 😁

**Pull request merged**